CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ This changelog adheres to [Keep a CHANGELOG](https://keepachangelog.com).
 
 ## [Unreleased]
 
+## Added
+-(PA-5326) Added macos 13 (x86_64)
+
 ### Added
```

Comment:
Can you append your change to the "Added" list that has already been started instead of creating a new "Added" list? Thanks